spec/README.md

```diff
@@ -16,51 +15,86 @@ The [EU U-Space law](https://eur-lex.europa.eu/legal-content/EN/TXT/?uri=CELEX%3
 
 ## A. Interface for Flight Authorization
-__Goal__: Allow the GCS to send flight plan to the USSP and get an associated status. 
+__Goal__: Allow the GCS to send a flight plan to the USSP and get an associated flight authorization of submission of that plan.
 
 __Pre-requisites__: 
 - A flight plan is built in the QGCS user-interface and it is valid 
 - A network connection exists between QGCS and USSP
+- Metadata e.g. Operator ID etc. exists that will be passed to the USSP system
 
 __Reference__: 
 - Example submission of a plan [as GeoJSON](https://github.com/openskies-sh/flight-blender/blob/master/api/flight-blender-1.0.0-resolved.yaml#L1402)
 - Example states of an operation [Flight Blender](https://github.com/openskies-sh/flight-blender/blob/master/api/flight-blender-1.0.0-resolved.yaml#L1885)
 
+__Deliverables__:
+- User interface in the QGCS panels to submit flight plan and get the responses
+
 ## B. Interface for Network RID / Telemetry
 
-__Goal__: Allow the GCS to send live telemetry (for a registered operation) to the USSP via this QGCS user-interace so that the USSP can process the data and share it with the UTM network. 
+__Goal__: Allow the GCS to send live telemetry (for a registered operation) to the USSP via this QGCS user-interface so that the USSP can process the data and share it with the UTM network. 
 __Pre-requisites__: 
+
 - A plan is built and shared to the USSP and authorized by the USSP (see Flight Authorization service below). 
 - Telemetry data from the drone should be submitted via TCP / IP with REST interfaces to the USSP that can be configured
 - Additional data e.g. operator location, drone serial number, operator registration number etc. 
 
+__Deliverables__:
+- Status indicators that telemetry is being submitted to the USSP (and the USSP is receiving it)
+- Ability to take remedial action incase of telemetry being interrupted
+
 __Reference__: 
 - Automated test for [ASTM network RID](https://github.com/interuss/monitoring/blob/main/monitoring/uss_qualifier/suites/astm/netrid/f3411_22a.yaml) exists
 - [Data format](https://github.com/uastech/standards/blob/dd4016b09fc8cb98f30c2a17b5a088fb2995ab54/remoteid/canonical.yaml#L1692) per ASTM Remote ID standard
 
 
 ## C. Interface for Geofence / Geozone
-Goal: To build a way for QGCS to read Geozone / Geofence information and show the data in the user interface
+__Goal__: To build a way for QGCS to read Geozone / Geofence information and show the latest data in the QGCS user interface. 
+
+__Pre-requisites__: 
+- In the EU this type of data is to be shared via the ED-269 / ED-26XX standard and provided via a API that the USSPs can automatically download these files / data periodically. This is out of scope of this project.
+- The data must be provided to the GCS via a API call and displayed on the user-interface.
+
+__Deliverables__:
+- Ability to show geo-zone information and its details on the QGCS user-interface
+
```

Comment:
propose another deliverable: Ability to explicitly acknowledge safety-critical geo-zones

Reply:
Thanks yes added